charts/argo-rollouts/templates/argo-rollouts-clusterrole.yaml

```diff
@@ -29,6 +29,14 @@ rules:
   - list
   - watch
   - patch
+- apiGroups:
+    - ""
```

Comment:
Should this be an empty list instead? ie
```yaml
 - apiGroups: {}
```

Reply:
from: https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-and-clusterrole

```yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: Role
metadata:
  namespace: default
  name: pod-reader
rules:
- apiGroups: [""] # "" indicates the core API group
  resources: ["pods"]
  verbs: ["get", "watch", "list"]
```